tools/REALTEK_RTL8195AM.py

```diff
@@ -0,0 +1,161 @@
+"""
+mbed REALTEK_RTL8195AM elf2bin script
+Copyright (c) 2011-2016 Realtek Semiconductor Corp.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+LIBRARIES BUILD
+"""
+
+import sys, array, struct, os, re, subprocess
+import hashlib
+
+from tools.paths import TOOLS_BOOTLOADERS
+from datetime import datetime
+
+# Constant Variables
+RAM2_RSVD = 0x3131373835393138
+
+def write_fixed_width_string (value, width, output):
```

Comment:
For consistency with the rest of the tools, could you remove the space in front of the `(`?